### PR TITLE
Release 1.11 preparations

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

**Comment by tribal-tec** [comment]
hold on for merge

**Comment by tribal-tec** [comment]
ping

File: CMakeLists.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@ list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake
 include(GitExternal)
 
 set(VERSION_MAJOR "1")
-set(VERSION_MINOR "12")
+set(VERSION_MINOR "11")
 set(VERSION_PATCH "0")
 set(VERSION_ABI 4)
```
Comment (eile) [inline CMakeLists.txt]: ABI unchanged? Servus separation not a ++major?

File: lunchbox/CMakeLists.txt [hunk 2]
```diff
@@ -3,7 +3,8 @@
 
 include(files.cmake)
 
-list(APPEND LUNCHBOX_LINK_LIBRARIES ${PTHREAD_LIBRARIES} ${Boost_LIBRARIES})
+list(APPEND LUNCHBOX_LINK_LIBRARIES
+  ${PTHREAD_LIBRARIES} ${Boost_LIBRARIES} ${SERVUS_LIBRARIES})
```
Comment (eile) [inline lunchbox/CMakeLists.txt]: o_O

File: lunchbox/CMakeLists.txt [hunk 3]
```diff
@@ -34,10 +35,6 @@ if(SKV_FOUND)
       ${PTHREAD_LIBRARIES})
   endif()
 endif()
-if(NOT LUNCHBOX_BUILD_V2_API)
-  list(APPEND LUNCHBOX_LINK_LIBRARIES ${SERVUS_LIBRARIES})
-endif()
```
Comment (eile) [inline lunchbox/CMakeLists.txt]: Ahh, so Servus is always required? Does LB use it directly?

File: lunchbox/bitOperation.h [hunk 4]
```diff
@@ -22,7 +22,7 @@
 
 #include <lunchbox/compiler.h>       // GCC version
 #include <lunchbox/types.h>
-#include <lunchbox/uint128_t.h>
+#include <servus/uint128_t.h>
```
Comment (eile) [inline lunchbox/bitOperation.h]: Yes. Ok then.

File: lunchbox/stdExt.h [hunk 5]
```diff
@@ -28,7 +28,11 @@
 
 #include <lunchbox/algorithm.h> // usort
 #include <lunchbox/compiler.h>
-#include <lunchbox/uint128_t.h>
+#ifdef LUNCHBOX_USE_V1_API
+#  include <lunchbox/uint128_t.h>
+#else
+#  include <servus/uint128_t.h>
+#endif
```
Comment (eile) [inline lunchbox/stdExt.h]: Only else path?

**Comment by eile** [comment]
+1 except for major version decision.

File: CMakeLists.txt (hunk 1 above)
Comment (tribal-tec) [inline CMakeLists.txt]: ABI from 1.10 was 3, so it is changed

File: lunchbox/CMakeLists.txt (hunk 3 above)
Comment (tribal-tec) [inline lunchbox/CMakeLists.txt]: Yes?! uint128_t in persistantmap,  requesthandler, plugin..., URI in peristantmap

File: lunchbox/stdExt.h (hunk 5 above)
Comment (tribal-tec) [inline lunchbox/stdExt.h]: you want elif V2_API?

File: CMakeLists.txt (hunk 1 above)
Comment (eile) [inline CMakeLists.txt]: Sorry, meant API. Didn't the servus and pression extraction change the API?

File: lunchbox/stdExt.h (hunk 5 above)
Comment (eile) [inline lunchbox/stdExt.h]: If major==2 (see above), then we only need the else path.

File: CMakeLists.txt (hunk 1 above)
Comment (tribal-tec) [inline CMakeLists.txt]: Of course they did, but the last release from Oct 14 was 1.11. Or you want to do even numbering?

**Pull request merged**